modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePartitionExchangeManager.java

```diff
@@ -160,6 +164,9 @@
     /** */
     private static final IgniteProductVersion EXCHANGE_PROTOCOL_2_SINCE = IgniteProductVersion.fromString("2.1.4");
 
+    /** Stripe id for cluster activation event. */
+    private static final int CLUSTER_ACTIVATION_EVT_STRIPE_ID = 1;
```

Comment:
Why 1? Why not 0?

Come to think of it, I will put something like (2<<31) - 1 here so we always land in the last stripe, which is probably least busy.